components/properties/src/maps.rs

```diff
@@ -96,6 +96,54 @@ where
     get_cp_map(provider, key::SCRIPT_V1)
 }
 
+/// Return a [`CodePointTrie`] for the East_Asian_Width Unicode enumerated
+/// property. See [`East_Asian_Width`].
+///
+/// # Example
+///
+/// ```
+/// use icu::properties::{maps, EastAsianWidth};
+///
+/// let provider = icu_testdata::get_provider();
+/// let payload = maps::get_east_asian_width(&provider).expect("The data should be valid!");
+/// let eaw = &payload.get().code_point_trie;
+///
+/// assert_eq!(eaw.get('ｱ' as u32), EastAsianWidth::Halfwidth); // U+FF71: Halfwidth Katakana Letter A
+/// assert_eq!(eaw.get('ア' as u32), EastAsianWidth::Wide); //U+30A2: Katakana Letter A
+/// ```
+///
+/// [`CodePointTrie`]: icu_codepointtrie::codepointtrie::CodePointTrie
```

Comment:
Nit, here and elsewhere: Please update these comments to point to `icu_codepointtrie::CodePointTrie`; I hoisted it to the top level

Reply:
Added a commit to address this.